### PR TITLE
The dropdown gets closed when same value is reselected

### LINKED ISSUE #46: Prevent item deselection
Currently, `iron-list` deselects items when you click on a selected item.

- Instead, we should just close the popup when user clicks on a selected item. `selected-items-changed` should NOT fire.

### PR DESCRIPTION
**Pull request by tomivirkki**
Fixes #46

[Review on Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box/56)

### CODE REVIEW

File: test/selecting-items.html [hunk 1]
```diff
@@ -25,36 +25,52 @@
     var combobox;
     var input;
     var selector;
+    var valueChangedSpy;
 
     beforeEach(function() {
       combobox = fixture('combobox');
-      input = combobox.$$("input");
+      combobox.items = ['foo', 'bar'];
```
Comment (jounik) [inline test/selecting-items.html]: Do we have some coding conventions about these, btw? Would be nice if everyone used the same (and if there was a linter that warned about using the incorrect ones).

**Comment by tehapo** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[test/selecting-items.html, line 32 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56#-K3yx3SRnVHpriG4KmO9-r1-32:-K4-XuHjNp9ARnVmxAb3:58164517) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/ad30db944026720d0c9f452707ab67b26c555b8b/test/selecting-items.html#L32)):*
I was looking into JSCS (http://jscs.info/), which looked nice, but the Atom plugin is missing the feature to check JS code inside an HTML page (https://github.com/AtomLinter/linter-jscs/issues/110).

The command-line version does have an `--extract` parameter and it could probably be pretty easily integrated into our build process.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56)*

**Comment by tehapo** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56)*

**Comment by tehapo** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[test/selecting-items.html, line 32 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56#-K3yx3SRnVHpriG4KmO9-r1-32:-K42QYThq5JFn48kcsOh:-1253192799) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/ad30db944026720d0c9f452707ab67b26c555b8b/test/selecting-items.html#L32)):*
Moved this discussion to a separate ticket.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56)*

**Comment by tehapo** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[test/selecting-items.html, line 32 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56#-K3yx3SRnVHpriG4KmO9-r1-32:-K42QhtBfXS8OFn5tjkB:-772607421) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/ad30db944026720d0c9f452707ab67b26c555b8b/test/selecting-items.html#L32)):*
https://github.com/vaadin/vaadin-elements/issues/25

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56)*

**Comment by tehapo** [comment]
LGTM.

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/56#-:-K42Qn4VGq22HK97Lfpe:-492975211)*

**Pull request merged**